pytorch/training/docker/1.11/py3/cu115/Dockerfile.gpu

```diff
@@ -124,10 +124,14 @@ RUN mkdir /tmp/efa \
  && curl -O https://s3-us-west-2.amazonaws.com/aws-efa-installer/aws-efa-installer-${EFA_VERSION}.tar.gz \
  && tar -xf aws-efa-installer-${EFA_VERSION}.tar.gz \
  && cd aws-efa-installer \
+ && apt-get update \
+ && apt-get upgrade -y \
  && ./efa_installer.sh -y --skip-kmod -g \
  && rm -rf $OPEN_MPI_PATH \
  && rm -rf /tmp/efa \
- && rm -rf /tmp/aws-efa-installer-${EFA_VERSION}.tar.gz
+ && rm -rf /tmp/aws-efa-installer-${EFA_VERSION}.tar.gz \
+ && rm -rf /var/lib/apt/lists/* \
+ && apt-get clean
```

Comment:
can you add below change to this file?

https://github.com/aws/deep-learning-containers/blob/master/pytorch/training/docker/1.10/py3/cu113/Dockerfile.e3.gpu#L199

Reply:
Discussed offline, already addressed.